h/views/main.py

```diff
@@ -87,7 +87,10 @@ def _html_link(request, annotation):
 
 def _incontext_link(request, annotation):
```

Comment:
I'd suggest you move everything link-related into its own module, say `h.links`, where `incontext` and `html` can become public functions, with their own tests. Testing the `includeme` is a route to sadness and despair :)